packages/consoledot-api/src/queries/index.ts

```diff
@@ -9,3 +9,4 @@ export * from "./useUserAccounts";
 export * from "./useConsumerGroup";
 export * from "./useAcls";
 export * from "./useServiceAccounts";
+export * from "./UseUser";
```

Comment:
```suggestion
export * from "./useUser";
```

The filename should have the same case as the exported function.